### ISSUE TITLE
[closetag] doesn't close tag when using Android Chrome.

### ISSUE THREAD

**Issue by takamori** [codemirror/CodeMirror#5461]
I just noticed this on the demo page. Just thought I'd mention it in case it wasn't expected/known; it doesn't cause explicit problems for our usage. Using Chrome on latest Android.

**Comment by marijnh** [comment]
Right, due to the way Android fires composition events, rather than keydown events, for text input via a virtual keyboard, handlers bound to keys that input text won't be run. In the rewrite we'll try to express such things in terms of text input events rather than specific keys, which should help.

**Comment by sanatankc** [comment]
Is there any way I can temporarily hack it to work for android?
